setup.py

```diff
@@ -47,27 +47,28 @@ def run(self):
 
 
 setup(
-    name='WoeUSB-ng',
+    name='woeusb-ng',
     version='0.2.10',
-    description='WoeUSB-ng is a simple tool that enable you to create your own usb stick windows installer from an iso image or a real DVD. This is a rewrite of original WoeUSB. ',
+    description='woeusb-ng is a simple tool that enable you to create your own usb stick windows installer from an iso image or a real DVD. This is a rewrite of original WoeUSB.',
     long_description=long_description,
     long_description_content_type='text/markdown',
     url='https://github.com/WoeUSB/WoeUSB-ng',
     author='Jakub Szymański',
     author_email='jakubmateusz@poczta.onet.pl',
-    license='GPL-3',
+    license='GPL-3+',
     zip_safe=False,
-    packages=['WoeUSB'],
     include_package_data=True,
-    scripts=[
-        'WoeUSB/woeusb',
+    data_files=[
+        ("share/applications", ["miscellaneous/WoeUSB-ng.desktop"]),
+        ("share/polkit-1/actions", ["miscellaneous/com.github.woeusb.woeusb-ng.policy"]),
+        ("share/icons/hicolor/scalable/apps", ["src/WoeUSB/data/woeusb-logo.png"]),
     ],
     install_requires=[
         'termcolor',
         'wxPython',
     ],
-    cmdclass={
-        'develop': PostDevelopCommand,
-        'install': PostInstallCommand
-    }
+    # cmdclass={
+    #     'install': PostInstallCommand,
+    #     'develop': PostDevelopCommand,
+    # }
```

Comment:
If the code is not needed simply drop them, we can always recover it from history when necessary.